MAUI/Scheduler/cell-selection.md

```diff
@@ -35,7 +35,7 @@ this.Scheduler.CellSelectionView.Stroke = Brush.Green;
 The cell selection background color can be customized by using the `Background` property in the `CellSelectionView`.
 The default value is Brush.Transparent.
```

Comment:
The default value is `Brush.Transparent`.

Reply:
Updated